### ISSUE TITLE
Pipe operator looks misaligned

### ISSUE THREAD

**Issue by point** [adobe-fonts/source-code-pro#200]
Some languages (like Elixir) has pipe operator |>
In source code pro it is not centered properly
![image](https://user-images.githubusercontent.com/121206/42695389-eb98f926-86bd-11e8-9c26-bed2ea2d4998.png)

**Comment by ianpan870102** [comment]
Yeah I also found that it seems to be purposely extending towards the bottom..?
